src/Http/Offer/AddImageRequestHandler.php

```diff
@@ -5,14 +5,16 @@
 namespace CultuurNet\UDB3\Http\Offer;
 
 use Broadway\CommandHandling\CommandBus;
-use CultuurNet\UDB3\Event\Commands\AddImage as EventAddImage;
-use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use Broadway\Repository\Repository;
+use CultuurNet\UDB3\Http\Organizer\AddMediaObjectPropertiesRequestBodyParser;
```

Comment:
This is a class in the `Organizer` namespace, so it should not be used in the `Offer` namespace. The request bodies are also different, on offers we only accept `mediaObjectId` at the moment it seems, and on organizers we accept `id`, `language`, `description`, `copyrightHolder` (on offers these can only be set afterwards with an update it seems). It would definitely be good to align those better in the future, but for now I would keep these separated in the code as two separate classes for the different endpoints / request bodies